docs/changelog.rst

```diff
@@ -1,7 +1,7 @@
 =========
 Changelog
 =========
-
+* :feature:`5046` Improve experience for users with old database when resolving assets.
```

Comment:
Add a test for this adding an asset that should be constant and fails to resolve so it queries it from the packaged globaldb. I believe is enough if you delete LUSD for example from the db